apps/meteor/ee/server/startup/broker.ts

```diff
@@ -10,7 +10,7 @@ const {
 	TRANSPORTER = '',
 	CACHE = 'Memory',
 	// SERIALIZER = 'MsgPack',
-	SERIALIZER = 'EJSON',
+	SERIALIZER = 'CBOR',
```

Comment:
Do this serializer works when the data is going through TCP transporter? Probably yes, but would like to confirm

Reply:
Yes, it does. Moleculer even separates these concepts as they can be agnostic of each other's details. But thanks for asking, I checked it anyway and it does work :wink: